README.md

```diff
@@ -36,9 +36,25 @@ this folder (in our build process) and integrated into the Jekyll Site. Follow
 the [local development guide](./docs/local-development.md) to get started with
 building the Jekyll site.
 
-### `/src-api` - API V2 Build Tooling
+`/src-api` - API V1 and V2 Build Tooling
 
-This is the build tooling we use for automatically generating documentation for
+Our API documentation source can be found in this folder.
+
+**API v1** is written by hand, and compiled to work with
```

Comment:
Should this be "v1.1"? I'm not sure, so I will defer to you, but just wanted to ask.

Reply:
I suppose it could be `1.x` but 1.1 works too.